client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -146,7 +148,10 @@ class CalypsoifyIframe extends Component< Props & ConnectedProps & ProtectedForm
 			this.iframeRef.current.contentWindow
 		) {
 			this.successfulIframeLoad = true;
-			const { port1: iframePortObject, port2: transferredPortObject } = new MessageChannel();
+			const {
+				port1: iframePortObject,
+				port2: transferredPortObject,
+			} = new globalThis.MessageChannel();
```

Comment:
`globalThis` is a code change here.

Reply:
How would you compare adding `globalThis`  vs. something like `/* global MessageChannel */` at the top of the file?

Im guessing there isn't any functional difference between the two and since we only have 1 instance of `MessageChannel` being called in the file it makes more sense to call it off the `globalThis` here explicitly?